test/e2e/cross_os/test_installer.py

```diff
@@ -0,0 +1,27 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+"""
+This test module contains tests that verify the Installer's behavior by submitting commands to the
+Deadline Cloud worker and checking that the result/output of the worker agent is as we expect it.
+"""
+import pytest
+import logging
+from deadline_test_fixtures import EC2InstanceWorker
+from utils import get_operating_system_name
+
+LOG = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize("operating_system", [get_operating_system_name()], indirect=True)
```

Comment:
If I'm reading this right, it looks like we change which operating system the worker is to be the same operating system of the machine we're submitting from. 

Looking at the test case, it looks like the test only works for linux. 

Have you tested submitting this to a windows worker?

Reply:
Changed to only run on Linux